### DISCUSSION TITLE
context loss

### DISCUSSION THREAD

**Discussion by developer239** [pmndrs/zustand#900] [Q&A]
In the main readme.md file you mention that Zustand is immune to `context loss` and then you link to 3-year-old issues that talks about even older library and the library is now deprecated. Isn't that a little misleading? 🤔

**Comment by dai-shi** [comment]
This is still true for libraries such as https://github.com/pmndrs/react-three-fiber, one of the most popular libraries at @pmndrs. 

(but, the link to `react concurrency` is incorrect now... 👁️ )

**Comment by dai-shi** [comment]
@drcmda any thoughts?
